### PR TITLE
Improve toggl reports format

### PR DESCRIPTION
**Pull request by mirland**

### CODE REVIEW

File: src/com/xmartlabs/slackbot/handlers/TogglReportSlashCommandHandler.kt [hunk 1]
```diff
@@ -4,25 +4,23 @@ import com.slack.api.bolt.context.builtin.SlashCommandContext
...
+        private const val NO_PERMISSION_MESSAGE = "You have no permission to do that."
     }
 
     override fun apply(req: SlashCommandRequest, ctx: SlashCommandContext): Response =
         if (UserSlackRepository.hasAdminPrivileges(req.payload.userId)) {
-            GlobalScope.launch {
-                TogglReportManager.sendWeeklyWorkingTimeReport(req.payload.userId, LocalDate.now().minusWeeks(1))
+            ctx.client().viewsOpen { r ->
+                r
+                    .triggerId(ctx.triggerId)
```
Comment (f7deleon) [inline src/com/xmartlabs/slackbot/handlers/TogglReportSlashCommandHandler.kt:19]: previous line?

File: src/com/xmartlabs/slackbot/manager/TogglReportManager.kt [hunk 2]
```diff
@@ -126,25 +116,51 @@ object TogglReportManager {
...
+        }
+        .formatReports(reportType)
 }
 
-enum class ReportType {
+@Suppress("MagicNumber")
+private fun List<TogglUserEntryReport>.formatReports(reportType: ReportType) = joinToString("\n") { entry ->
+    val (hours, minutes) = entry.duration(reportType)
+        .let { workTime -> workTime.toHours() to workTime.toMinutesPart() }
+    "   • ${entry.togglUser.name}, $hours:${if (minutes < 10) "0$minutes" else minutes}"
```
Comment (f7deleon) [inline src/com/xmartlabs/slackbot/manager/TogglReportManager.kt:139]: what about an extension for the minute's format?

**Review by f7deleon — APPROVED** [review]

File: src/com/xmartlabs/slackbot/manager/TogglReportManager.kt (hunk 2 above)
Comment (mirland) [reply src/com/xmartlabs/slackbot/manager/TogglReportManager.kt:139]: I did it, I will open a new pr with that

File: src/com/xmartlabs/slackbot/handlers/TogglReportSlashCommandHandler.kt (hunk 1 above)
Comment (mirland) [reply src/com/xmartlabs/slackbot/handlers/TogglReportSlashCommandHandler.kt:19]: The auto format changes the line, I will rename the variable

**Pull request merged**